components/MhiAcCtrl/mhi_ac_ctrl.h

```diff
@@ -150,9 +152,9 @@ class MhiAcCtrl : public climate::Climate,
     }
     void loop() override
     {
-        if(millis() - room_temp_api_timeout_ms >= room_temp_api_timeout*1000) {
+        if(esphome::millis() - room_temp_api_timeout_ms >= room_temp_api_timeout*1000) {
```

Comment:
You should not have to prefix every call to `millis()` with the `esphome::` prefix while already within the esphome namespace.